fuel-core-interfaces/src/bft.rs

```diff
@@ -0,0 +1,17 @@
+use super::model::{FuelBlock, SealedFuelBlock};
+use anyhow::Result;
+use std::sync::Arc;
+use tokio::sync::oneshot;
+
+pub enum BFTMpsc {
+    CheckBlockConsensus {
+        block: Arc<SealedFuelBlock>,
+        ret: oneshot::Sender<Result<()>>,
+    },
+    CheckBlockLeader {
+        block: Arc<FuelBlock>,
```

Comment:
I just noticed that we don't have the total committed rounds as part of the block header. We need to add it because its a requirement for bft to calculate the current round; also provided the timestamp.

For more info:
https://github.com/FuelLabs/fuel-bft/blob/7acc238ee02ec8cbc726deb97519b075fc0ab538/src/reactor.rs#L67-L79

Reply:
This should be added, can you create a github issue for it as it is not related to this pull request.

Reply:
tracked: https://github.com/FuelLabs/fuel-core/issues/399